### DISCUSSION TITLE
CSS Not being applied

### DISCUSSION THREAD

**Discussion by Andy-Bracket** [community/community#22710] [Other Feature Feedback, Questions, & Ideas]
<p>Hi Github,</p>
<p>Very new to all of this so forgive me, but my CSS is not being applied to my html page, most of the other topic’s I’ve read all point to a referencing issue, however I have tried a few things and a few different references with no luck, I am certain I am missing something obvious.</p>
<p>Link: <a href="https://github.com/Mandrake-ux/First-Site" rel="nofollow noopener">https://github.com/Mandrake-ux/First-Site</a></p>
<p>Any assistance will be appreciated.</p>

**Comment by SethClydesdale** [comment]
<p>Hello <a class="mention" href="/mandrake-ux">@Mandrake-ux</a>, welcome to the GitHub Support Community!</p>
<p>The issue appears to be related to the capitalization of the directory. Your css stylesheet currently resides in the <strong>CSS</strong> (uppercase) directory, however, in your webpage you’re referencing the <strong>css</strong> (lowercase) directory.</p>
<p>The directories are case-sensitive, so the issue can be solved in one of two ways.</p>
<ol>
<li>rename the CSS folder --&gt; css</li>
<li>or change the referenced directory from css --&gt; CSS</li>
</ol>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/f2a0c4cb0e049f2b6d19ead2c508c0bd828713c5.png" data-download-href="/uploads/short-url/yCnZA3gCVuH3gSDVdltZc3PpvUN.png?dl=1" title="css" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10122549/181072668-a9b09392-6a8a-4504-a08b-44bf82ffd224.png" alt="css" data-base62-sha1="yCnZA3gCVuH3gSDVdltZc3PpvUN" width="690" height="276" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/f2a0c4cb0e049f2b6d19ead2c508c0bd828713c5_2_690x276.png, https://aws1.discourse-cdn.com/github/optimized/2X/f/f2a0c4cb0e049f2b6d19ead2c508c0bd828713c5_2_1035x414.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/f/f2a0c4cb0e049f2b6d19ead2c508c0bd828713c5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/f2a0c4cb0e049f2b6d19ead2c508c0bd828713c5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">css</span><span class="informations">1316×527 35.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by tajmone** [comment, marked as answer]
<p>Hi <a class="mention" href="/mandrake-ux">@Mandrake-ux</a> and welcome on board!</p>
<p><a class="mention" href="/SethClydesdale">@SethClydesdale</a> is right, GH servers being Linux based have a case-sensitive file system.</p>
<p>Since you didn’t spot the problem locally, I’m assuming you’re working under Windows OS.</p>
<p>With Windows 10 you can now enable case-sensitive behavior on  a per-directory basis, which is always a good solution when working with cross-platform projects on GitHub:</p>
<ul>
<li><a href="https://www.howtogeek.com/354220/how-to-enable-case-sensitive-folders-on-windows-10/" rel="nofollow noopener">How-To-Geek » How to Enable Case Sensitive File and Folder Names on Windows 10</a></li>
</ul>
<p>I recommend you apply case-sensitiveness to your local GitHub repositories — or store all repositories in a common root folder on which you enable case-sensitiveness — which would allow you to (1) spot case-related issues like this one, (2) prevent cross-platform inadvertent issues, (3) allow you to work with repositories that contain same-named file with different letter casing.</p>
<p>Just be aware that enabling case-sensitive files under Windows might lead to inconsistent behavior with some editors and IDEs, which might not detect correctly name changes or be able to handle same-named files with different casing (you’ll need to test various editors for this).</p>

**Comment by Andy-Bracket** [comment]
<p>Hello <a class="mention" href="/SethClydesdale">@SethClydesdale</a>, Thank you so much that fixed the issue, I had no idea they were case sensitive, thank you for the info and help!</p>

**Comment by Andy-Bracket** [comment]
<p>Hello <a class="mention" href="/tajmone">@tajmone</a> Thank you so much for the feedback.</p>
<p>I really appreciate the detailed response, and have learned some helpful new things in my journey today. You are correct, I am indeed working with Windows 10. Thank you for the link, I will definitely be following this practice in future!</p>
